src/integrationTest/java/uk/gov/hmcts/reform/wataskmanagementapi/services/DatabaseReplicationCheckTest.java

```diff
@@ -0,0 +1,170 @@
+
+        await().ignoreException(AssertionFailedError.class)
+            .pollInterval(1, SECONDS)
+            .atMost(30, SECONDS)
+            .until(
+                () -> {
+                    boolean columnAdded = checkIfColumnAddedInTable(TASKS_TABLE_NAME,
+                                                                    TERMINATION_PROCESS_COLUMN_NAME,
+                                                                    container);
+                    assertTrue(columnAdded);
+
+                    columnAdded = checkIfColumnAddedInTable(TASKS_TABLE_NAME,
+                                                            TERMINATION_PROCESS_COLUMN_NAME,
+                                                            containerReplica);
+                    assertTrue(columnAdded);
+                    columnAdded = checkIfColumnAddedInTable(TASK_HISTORY_TABLE_NAME,
+                                                            TERMINATION_PROCESS_COLUMN_NAME,
+                                                            containerReplica);
+                    assertTrue(columnAdded);
+                    columnAdded = checkIfColumnAddedInTable(REPORTABLE_TASK_TABLE_NAME,
+                                                            TERMINATION_PROCESS_COLUMN_NAME,
+                                                            containerReplica);
+                    assertTrue(columnAdded);
+                    return true;
+                });
+    }
+
+    private String selectTaskColumnValueFromTable(String taskId, String tableName, String columnName,
+                                                  JdbcDatabaseContainer container) {
+        String selectQuery = "select " + columnName + " from cft_task_db." + tableName + " where task_id = ? ";
+        String columnValue = null;
+        try (Connection conn = DriverManager.getConnection(container.getJdbcUrl(),
+                                                            container.getUsername(),
+                                                            container.getPassword());
+             PreparedStatement preparedStatement = conn.prepareStatement(selectQuery)) {
+            preparedStatement.setString(1, taskId);
+            boolean hasResultSet = preparedStatement.execute();
+            if (hasResultSet) {
+                log.info("Task found in the replica table");
+                ResultSet rs = preparedStatement.getResultSet();
+                while (rs.next()) {
+                    columnValue = rs.getString(columnName);
+                }
+            } else {
+                log.error("Task not found in the replica table");
+            }
+
+        } catch (Exception e) {
+            log.error("select query to get " + columnName + " failed  : {}, {} ",
+                      e.getCause(), e.getMessage()
+            );
+        }
+        return columnValue;
+    }
+
+    private boolean checkIfColumnAddedInTable(String tableName, String columnName,
```

Comment:
Should be able to use JPA for this type of thing as suggested above?
The `container` was only used for some admin set up stuff which I don't think you need here.  You ought to have an existing DB connection within the Spring test context.

Reply:
As discussed, we will do this in the follow up PR when we have the entity code.